.env.sample

```diff
@@ -21,3 +21,7 @@ DISABLE_CRONS=false
 # Following distances in blocks to guarantee finality on each chain. Format should be a map chainId -> blocks.
 # E.g. { "137": 100 }
 FOLLOWING_DISTANCES=
+JWT_SECRET=secret
+DISCORD_CLIENT_ID=
+DISCORD_CLIENT_SECRET=
+DISCORD_REDIRECT_URI=
```

Comment:
Similar to other fields, it might be useful to either provide a comment or sample value for these.